Packs/DemistoRESTAPI/Scripts/GetIncidentTasks/GetIncidentTasks_test.py

```diff
@@ -0,0 +1,100 @@
+                        "type": "regular"
+                    },
+                    "type": "regular"
+                }
+            }
+        },
+        "task": {
+            "name": "Process Sub-playbook",
+            "type": "playbook",
+            "version": 8
+        },
+        "type": "playbook"
+    }
+}
+
+
+@pytest.mark.parametrize('name, tag, states, output', [('test', None, 'Completed',
+                                                        'Tasks with name "test" and states "Completed"'),
+                                                       (None, None, 'Completed,Skipped',
+                                                        'Tasks with states "Completed,Skipped"'),
+                                                       (None, 'testtag', None, 'Tasks with tag "testtag"')])
+def test_format_title(name, tag, states, output):
+    assert format_title(name, tag, states) == output
+
+
+@pytest.mark.parametrize('task, name, tag, states, output', [(SAMPLE_TASKS['1'],
+                                                              'Example Task Name', None, ['Completed'], True),
+                                                             (SAMPLE_TASKS['1'],
+                                                              None, 'testtag', ['Completed'], True),
+                                                             (SAMPLE_TASKS['2'],
+                                                              '', 'testtag', ['Completed'], False),
+                                                             (SAMPLE_TASKS['1'], None, 'testtag', [], True)])
+def test_is_task_match(task, name, tag, states, output):
+    assert is_task_match(task, name, tag, states) == output
+
+
+@pytest.mark.parametrize('states, output', [(['Completed'], ['Completed']),
+                                            ([], ['', 'inprogress', 'Completed', 'Waiting', 'Error',
+                                                  'LoopError', 'WillNotBeExecuted', 'Blocked']),
+                                            (['error'], ['Error', 'LoopError'])])
+def test_get_states(states, output):
+    assert get_states(states) == output
+
+
+@pytest.mark.parametrize('tasks, output', [([SAMPLE_TASKS['1']], [SAMPLE_TASKS['1']]),
+                                           ([SAMPLE_TASKS['3']], [SAMPLE_TASKS['3']['subPlaybook']['tasks']['4'],
+                                                                  SAMPLE_TASKS['3']]),
+                                           ([], [])])
+def test_get_playbook_tasks(tasks, output):
+    assert get_playbook_tasks(tasks) == output
+
+
+@pytest.mark.parametrize('args', [({'name': 'task name'})])
+def test_get_task_command_missing(args):
+    with pytest.raises(KeyError):
+        get_task_command(args)
+
+
+@pytest.mark.parametrize('args', [({'inc_id': '1', 'name': 'task name'})])
+def test_get_task_command_failure(args):
+    with pytest.raises(Exception):
+        get_task_command(args)
```

Comment:
Please note that there is no need to test the `args`. The `args` have already been tested, and the server should always return a dictionary. 

Could you please create a unit test for the `get_task_command` function and use the `mocker` fixture to mock the response from the `demisto.executeCommand('core-api-get', {'uri': f'/investigation/{inc_id}/workplan'})` command? 

Here's an [example](https://github.com/demisto/content/blob/master/Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG_test.py#L58) of how to use `mocker` to patch the `executeCommand` function:

```python
mocker.patch.object(demisto, "executeCommand", return_value=INCIDENT_TASK)
```